bauble/src/parse/copy.rs

```diff
@@ -0,0 +1,59 @@
+use indexmap::IndexMap;
+
+use crate::parse::PathEnd;
+
+use super::{Ident, Object, Value};
+
+// TEMP
+pub fn copy(values: Vec<&mut Value>, copies: &IndexMap<Ident, Object>) {
```

Comment:
What is this for?

Reply:
I originally wrote this function to resolve objects that reference `copy` objects, just to get it working. If it's not being used anymore, it can be removed